core/res/res/values/config.xml

```diff
@@ -1646,6 +1646,8 @@
     <string-array name="config_locationProviderPackageNames" translatable="false">
         <!-- The standard AOSP fused location provider -->
         <item>com.android.location.fused</item>
+        <!-- The (faked) microg fused location provider (a free reimplementation) -->
```

Comment:
We should start without this and consider it later on. I want to start without granting it any special privileges, and we want to provide supplementary location services ourselves within GrapheneOS.